pkg/hostagent/hostagent.go

```diff
@@ -732,7 +733,7 @@ func forwardSSH(ctx context.Context, sshConfig *ssh.SSHConfig, port int, local,
 				}
 			} else {
 				logrus.WithError(err).Warnf("Failed to set up forward from %q (guest) to %q (host)", remote, local)
-				if removeErr := os.RemoveAll(local); err != nil {
```

Comment:
```
pkg/hostagent/hostagent.go:735:8: uncheckedInlineErr: removeErr error is unchecked, maybe intended to check it instead of err (gocritic)
                                if removeErr := os.RemoveAll(local); err != nil {
                                   ^
```